### PR TITLE
Fixed hyperlinks

### LINKED ISSUE #231: ReleaseFix for #199
Compiles fine in MkDocs.

We noted the following 4 errors in the wiki page, which should be simple to fix (See image screenshots below and corresponding text):

**Asterixs appear, do to improper use of syntax**
errorOneWiki199

**Missing hyperlinks** ( Use: \[Text](URL) )
Two example images are shown below, there are several more in the text.
ErrorTwoURLLinks1

ErrorTwoURLLinks2

**Improper use of listings, should be in proper list format on new lines**
ErrorThreeImproperListing

**Improper use of listings, should be in proper list format on new lines, same as above, and note that the first list item is on the same line as the colon**
ErrorFourImproperListingAndNewLine

### PR DESCRIPTION
**Pull request by MrPekar98**
fixes #231

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw607f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw618f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

File: docs/development/Offline_mode.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-**Flutter resources:**    
-https://github.com/tekartik/sqflite/blob/master/sqflite/README.md  
-https://pub.dev/packages/connectivity   
-https://flutter.dev/docs/cookbook/images/cached-images   
-https://pub.dev/packages/cached_network_image
+**Flutter resources:**
+
+[SQFlite](https://github.com/tekartik/sqflite/blob/master/sqflite/README.md)  
+
+[connectivity](https://pub.dev/packages/connectivity)
```
Comment (Jakobzs) [inline docs/development/Offline_mode.md:7]:
```suggestion
[Connectivity](https://pub.dev/packages/connectivity)

```

File: docs/development/Offline_mode.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+
+[Cached Images](https://flutter.dev/docs/cookbook/images/cached-images)
+
+[cached_network_image](https://pub.dev/packages/cached_network_image)
```
Comment (Jakobzs) [inline docs/development/Offline_mode.md:11]:
```suggestion
[Cached Network Image](https://pub.dev/packages/cached_network_image)

```

**Review by rasmus91 — APPROVED** [review]

**Review by Jakobzs — CHANGES_REQUESTED** [review]

File: docs/development/Offline_mode.md (hunk 1 above)
Comment (MrPekar98) [reply docs/development/Offline_mode.md:7]: It's the name of the package. You still think is should be with capital 'C'?

Comment (Jakobzs) [reply docs/development/Offline_mode.md:7]: No, it is fine then.

File: docs/development/Offline_mode.md (hunk 2 above)
Comment (Jakobzs) [reply docs/development/Offline_mode.md:11]: Ignore this one as well, package name.

**Review by Jakobzs — APPROVED** [review]

**Pull request merged**